src/connectors/scene/CryptoExchangeSceneConnector.js

```diff
@@ -8,13 +8,13 @@ import { selectWalletForExchange } from '../../actions/CryptoExchangeActions.js'
 import * as actions from '../../actions/indexActions'
 import * as Constants from '../../constants/indexConstants'
 import s from '../../locales/strings.js'
-import { getExchangeRate } from '../../modules/Core/selectors.js'
 import type { Dispatch, State } from '../../modules/ReduxTypes'
 import { CryptoExchangeSceneComponent } from '../../modules/UI/scenes/CryptoExchange/CryptoExchangeSceneComponent'
 import type {
   CryptoExchangeSceneComponentDispatchProps,
   CryptoExchangeSceneComponentStateProps
 } from '../../modules/UI/scenes/CryptoExchange/CryptoExchangeSceneComponent'
+import { getExchangeRate } from '../../modules/UI/selectors.js'
 import { emptyCurrencyInfo, emptyGuiWallet } from '../../types.js'
```

Comment:
Here we are moving the `getExchangeRate` method from the core area to the UI since the exchange rates will now be kept in Redux in the GUI